config/reader.go

```diff
@@ -566,11 +566,15 @@ func getFromCloudGRPC(ctx context.Context, cloudCfg *Cloud, logger golog.Logger)
 
 	service := apppb.NewRobotServiceClient(conn)
 	res, err := service.Config(ctx, &apppb.ConfigRequest{Id: cloudCfg.ID, AgentInfo: agentInfo})
+
+	if res.GetSkippedErrors() != "" {
+		logger.Warnw("found errors in config that were skipped", "err", res.GetSkippedErrors())
+	}
```

Comment:
[nil] This should come after checking the err from `service.Config` otherwise res could be nil / empty and result in a segfault